components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -12886,8 +12894,15 @@ private void setCommonParametersForPolicy(PreparedStatement policyStatement, Pol
         } else {
             policyStatement.setString(11, UUID.randomUUID().toString());
         }
+
     }
 
+    private void setRateLimitDetails(ApplicationPolicy policy, ResultSet resultSet) throws SQLException {
+        if(resultSet.getInt(ThrottlePolicyConstants.COLUMN_RATE_LIMIT_COUNT) > 0) {
```

Comment:
format

Reply:
fixed in b56bf3020abe507022f90c1b46e64547b10cf7ac